### DISCUSSION TITLE
avarage FPS

### DISCUSSION THREAD

**Discussion by narayanls** [benjamimgois/goverlay#26] [Q&A] [answered]
Hello! first, congrats on the project.
I want to know if is possible to set Avg FPS on Goverlay for comparsions purporse on games.

Thank you for your time!

**Comment by benjamimgois** [comment, marked as answer]
Hi @narayanls , goverlay only implements the features available on Mangohud. Currently the only method available to display the FPS average is recording a logging (Left Shift + F2)

![image](https://user-images.githubusercontent.com/104206575/196723056-8c5c1a1f-0ac8-4d50-8d71-23c36a394ea2.png)

Thanks for the kind words
